pkg/cmd/utils/cli_client.go

```diff
@@ -35,9 +35,15 @@ func GetCliOutput() io.Writer {
 }
 
 //setGlobalClient creates cli's controller client from viper's configured serverIp
-func setGlobalClient() {
+func SetGlobalClient() {
 	url := GetConfiguredServerIP()
```

Comment:
I didn't understood why make it a exported func, since it doesn't seem to be used by other pkgs.

Reply:
About this just be made clearer, by making the first letter of a function uppercase it can be used by other people when importing the pkg. So one could do:
```go
import "inspr.dev/inspr/pkg/cmd/utils"

// using cli exported func
utils.SetGlobalClient()
```

But if the function is not being planned to be used by files in another pkg there is no necessity to make it exported